python-code/set-sample/sample.py

```diff
@@ -1,3 +1,5 @@
 VALUE1 = 2.0
 import os
```

Comment:
This is a Unusedimportos
```suggestion 
```

 from our GitHub Enhacement Bot